lib/std/rand.zig

```diff
@@ -61,14 +61,30 @@ pub const Random = struct {
     }
 
     /// Returns a random value from an enum, evenly distributed.
```

Comment:
Can we explain the differences between the two options? Something like this?
```suggestion
    /// Returns a random value from an enum, evenly distributed.
    /// Note that this will not yield consistent results across all targets
    /// due to pointer size differences.
    /// For that, specify an integer type of a fixed bit size to `enumValueWithIndex`.
```
Same for the other one.